src/v/container/fragmented_vector.h

```diff
@@ -480,6 +480,15 @@ class fragmented_vector {
      */
     void erase_to_end(const_iterator begin) { pop_back_n(cend() - begin); }
 
+    template<typename Range>
+    requires std::ranges::range<Range>
+    static fragmented_vector from(Range range) {
```

Comment:
What do you think of using `std::from_range_t` constructor parameter instead of a static method? This would keep the API more compatible with `std::vector`.

Reply:
i will change it for constructor, but `std::from_range_t` is available since C++23